jloadr-bootstrap/src/main/java/de/adito/jloadr/bootstrap/Starter.java

```diff
@@ -31,6 +29,11 @@ public static void main(String[] args) throws Throwable
     Throwable loadError = null;
     try
     {
+      String minimalVersion = System.getProperty("min.java");
```

Comment:
When there are checks for the runtime these things should be the first steps in executing a program. As with the arguments that are mandatory checking the java version should be done in the beginning before initialisation.